src/network/MissionControlProtocol.cpp

```diff
@@ -228,8 +228,8 @@ static bool validateJointPositionRequest(const json& j) {
 static void handleJointPositionRequest(const json& j) {
 	// TODO: ignore this message if we are in autonomous mode.
 	std::string motor = j["joint"];
-	double position_deg = j["position"];
-	int32_t position_mdeg = std::round(position_deg * 1000);
+	[[maybe_unused]] double position_deg = j["position"];
+	[[maybe_unused]] int32_t position_mdeg = std::round(position_deg * 1000);
```

Comment:
Instead of adding these hints, we can just comment out the body of this method and leave a `TODO` to implement at some point